tools/js-tools/eslintrc/base.mjs

```diff
@@ -220,6 +233,23 @@ export function makeBaseConfig( configurl, opts = {} ) {
 					},
 				],
 
+				'import/no-extraneous-dependencies': [
+					'error',
+					{
+						peerDependencies: true,
+					},
+				],
+				'import/no-unresolved': [
+					'error',
+					{
+						ignore: [
+							// Jest dummy package.
+							'^@jest/globals$',
+						],
+					},
+				],
+				'import/default': 'warn',
+				'import/named': 'warn',
```

Comment:
What are the implications of a warn vs. an error?

Reply:
For the monorepo, not much. If you run eslint manually it shows up as a yellow "warning" type instead of a red "error" type, that's about it.